test/vt_flash.jl

```diff
@@ -0,0 +1,155 @@
+using CubicEoS
```

Comment:
this file to large, should be splitted or even removed

Reply:
Perhaps some of functions here can be unit tests. The latter could introspection scripts.